ntl.c

```diff
@@ -148,11 +148,11 @@ ntl_to_buf(char *buf, size_t size, void **p, struct ntl_str_delimiter * d,
   return tsize;
 }
 
-int
+size_t
 ntl_to_abuf(char ** buf_p, void **p, struct ntl_str_delimiter * d,
```

Comment:
this one should be int as well as the value could be negative just as `asprintf`